lib/plugins/physics.js

```diff
@@ -34,7 +34,8 @@ function inject (bot) {
     left: false,
     right: false,
     jump: false,
-    sprint: false
+    sprint: false,
+    sneak: false,
```

Comment:
eslint doesn't like the trailing commas, you will need to remove it to confirm to the code style rules.